docs/source/life-cycle.rst

```diff
@@ -27,6 +27,8 @@ its critical dates, and which type of support it's currently eligible for.
 +------------+----------------+-----------------------+------------------+-------------------+-------------------------+
 | Version    | Status         | Changelogs            | GA Date          | EOL Date          | Life Cycle Phase        |
 +============+================+=======================+==================+===================+=========================+
+| 1.16.x     | Released       | `1.16.x changelogs`_  | TBD              | TBD               | `In preview`            |
```

Comment:
I don't know if this was in the other cherry pick or not... isn't 1.16.x status=Test or Pending or Bete?